knees4bees/whats-cookin
# Feature/implement grid

--->  ((are you in the right branch??))
## What does this PR do?
 - feature work to add grid to all components and test that mobile functionality works the same
## How should it be tested?
 - fetch the branch and run in your browser; you should see colors behind each section, which will be removed later, but they're super helpful in determining what section is where.
